sql/chunk.sql

```diff
@@ -65,6 +65,57 @@ BEGIN
 END
 $BODY$;
 
+CREATE OR REPLACE FUNCTION _timescaledb_internal.dimension_calculate_default_range_closed(
+        dimension_value   BIGINT,
+        num_slices        SMALLINT,
+    OUT range_start  BIGINT,
+    OUT range_end    BIGINT)
+LANGUAGE PLPGSQL STABLE AS
+$BODY$
+DECLARE
+    inter BIGINT;
+BEGIN
+       IF dimension_value < 0 THEN 
+            RAISE 'Dimension values for close dimensions should be positive. Got: %', dimension_value;
+       END IF;
+       inter := (2147483647 / num_slices);
```

Comment:
Is there really no way to define constants in PLPGSQL? I would really like to get rid of these fixed numbers spread all over the code. It is often unclear what they mean and you have to manually ensure they make sense and have the right value.